component-catalog/src/Examples/Carousel.elm

```diff
@@ -44,15 +45,32 @@ type alias Settings =
     { items : Int
     , controlListStyles : ( String, List Style )
     , controlStyles : ( String, Bool -> List Style )
+    , carouselType : CarouselType
     }
 
 
+type CarouselType
```

Comment:
I didn't check in detail so there may be other things missing but just FYI the code snippet for combined controls actually has a call to `viewWithTabControls`:

<img width="887" alt="Screenshot 2023-08-25 at 15 34 32" src="https://github.com/NoRedInk/noredink-ui/assets/753421/e3311ee2-2409-44ad-9ddc-cb468bcaddbb">

Totally fine to leave this all for the end of the code review once we have an API we are all happy with :-)